app/components/dashboard/in_progress_collection_row_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<div style="margin-bottom: 0.5rem;" class="row">
```

Comment:
adding some extra margin to these rows, else the complete draft buttons in each row will stack up too tightly

Reply:
can you use the `mb-*` bootstrap classes for this?